lib/lighthouse/letters_generator/client.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+require 'lighthouse/letters_generator/configuration'
+
+module Lighthouse
+  module LettersGenerator
+    class Client < Common::Client::Base
+      LETTER_TYPES = %w[
+        benefit_summary
+        benefit_summary_dependent
+        benefit_verification
+        certificate_of_eligibility
+        civil_service
+        commissary
+        medicare_partd
+        minimum_essential_coverage
+        proof_of_service
+        service_verification
+      ].to_set.freeze
+
+      configuration Lighthouse::LettersGenerator::Configuration
+
+      def initialize(conn = nil)
```

Comment:
Is there any reason to pass in a connection here other than for testing?

Reply:
Nope. What's your recommendation?

Reply:
I was curious if there was an issue with using VCR cassettes for testing. It seems like a bit of a code smell to incorporate custom connections just for testing

Reply:
Posting here for closure - our chats led to realizing that doing it this way has opened up an opportunity for making these types of services more general purpose (see https://github.com/department-of-veterans-affairs/vets-api/pull/12199/files)